cluster/addons/addon-manager/kube-addons.sh

```diff
@@ -106,7 +106,7 @@ function log() {
 
 # Generate kubectl prune-whitelist flags from provided resource list.
 function generate_prune_whitelist_flags() {
-  local -r resources=($@)
+  local -r resources=('$@')
```

Comment:
this change looks suspect 🤔

Reply:
what shellcheck failure is this addressing?